### PR TITLE
Ensure addon works with Ember 2.1+

### PR DESCRIPTION
**Pull request by sandersky**
# PATCH
# CHANGELOG
- **Fixed** addon so it works with older versions of Ember all the way back to Ember `2.1`.

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -7,6 +7,105 @@ module.exports = {
       }
     },
     {
+      name: 'ember-1-12',
```
Comment (rox163) [inline config/ember-try.js:10]: you are just adding these props but the build is just tested against 2.1 right?

Comment (sandersky) [reply config/ember-try.js:10]: Correct, I would have Travis test against all of them but due to Travis limiting us to 4 concurrent builds I'm not having CI run them all. By having them all defined here though a dev can easily run tests against a specific version via `ember try:one ember-2-3`.

Comment (sandersky) [reply config/ember-try.js:10]: I should note this is the same config I'm using elsewhere, i.e. `ember-prop-types`. I'm just copy/pasting it into other addons.

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/176/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**